### PR TITLE
Fix customMaterial waiting for shaders to be loaded

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

File: packages/dev/materials/src/custom/customMaterial.ts [hunk 1]
```diff
@@ -10,6 +10,9 @@ import { Color3, Color4 } from "core/Maths/math.color";
 import type { Nullable } from "core/types";
 import type { SubMesh } from "core/Meshes/subMesh";
 
+import "@babylonjs/core/Shaders/default.vertex";
+import "@babylonjs/core/Shaders/default.fragment";
```
Comment (ryantrem) [inline packages/dev/materials/src/custom/customMaterial.ts:14]: Why is this not
```suggestion
import "core/Shaders/default.vertex";
import "core/Shaders/default.fragment";

```

Comment (deltakosh) [reply packages/dev/materials/src/custom/customMaterial.ts]: Because @RaananW told me so :D

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Review by sebavan — APPROVED** [review]

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15629/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15629/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15629/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15629/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15629/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15629/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15629/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15629/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15629/merge/testResults/webgpuplaywright/index.html

File: packages/dev/materials/src/custom/customMaterial.ts (hunk 1 above)
Comment (RaananW) [reply packages/dev/materials/src/custom/customMaterial.ts]: Just noticed this comment. this was for the user, not for us in core :-)

**Pull request merged**